app/service/HeatMapSourceGenerator.js

```diff
@@ -135,8 +173,7 @@ angular
             var params = {
                 "q.text": keyword,
                 "q.time": '[' + this.getFormattedDateString(reqParamsUi.minDate) + ' TO ' + this.getFormattedDateString(reqParamsUi.maxDate) + ']',
-                "q.geo": '[' + bounds.minX + ',' + bounds.minY + ' TO ' + bounds.maxX + ',' + bounds.maxY + ']',
-                "a.hm.limit": 1000
+                "q.geo": '[' + bounds.minX + ',' + bounds.minY + ' TO ' + bounds.maxX + ',' + bounds.maxY + ']'
```

Comment:
No, this limitation will be only used for the search.

According to [API](http://54.158.101.33:8080/bopws/swagger/#!/default/export) csv export doesn't have this parameter.